src/napari_manual_transforms/_model.py

```diff
@@ -177,11 +211,19 @@ def origin(self, v) -> None:
     @property
     def transform(self) -> np.ndarray:
         M = np.eye(4)
-        M[:3, :3] = self.matrix
+        M[:3, :3] = self.matrix @ np.diag((max(self.scale, MINIMUM_SCALE),) * 3)
```

Comment:
seems like this `max()` could go in the setter... along with a warning if someone tries to set scale to 0.

Reply:
This should be resolved as validation now happens during the scale-setter.